teps/0092-scheduling-timeout.md

```diff
@@ -211,17 +213,17 @@ field would be deprecated once this feature is beta.
 ```yaml
 spec:
   timeouts:
-    scheduling: "5m" # this is the new option
-    execution: "10m" # this option would have the same functionality as the existing `timeout` field
+    scheduling: "5m" # optional, if specified, from startTime to scheduledTime
+    execution: "5m" # optional, if specified, from scheduledTime to completionTime
+    total: "10m" # this option would have the same functionality as the existing `timeout` field
 ```
 
 When set, the order of timeout application would be:
-1. At the creation of the `TaskRun`, the `scheduling` timeout counter would begin
+1. At the start of the `TaskRun`, the `scheduling` timeout counter would begin
 2. Once the pod executing the `TaskRun` has been scheduled, the `scheduling` timeout would stop and the `execution`
-   timeout would begin 
-   
-When the `scheduling` timeout is not set for a `TaskRun`, the `execution` timeout counter would begin as it currently
-does (from the `startTime` of the `TaskRun`).
+   timeout would begin
+When the `scheduling` timeout is not set for a `TaskRun`, the `execution` timeout counter would begin from the `startTime` of the `TaskRun`.
```

Comment:
This is no longer true; the execution timeout doesn't change meaning depending on whether the scheduling timeout is set.